pkg/updatechecker/updatechecker.go

```diff
@@ -243,19 +251,29 @@ func CheckForUpdates(appID string, deploy bool, skipPreflights bool, isCLI bool)
 		}
 
 		latestVersion := allVersions[len(allVersions)-1]
-		downstreams, err := store.GetStore().ListDownstreamsForApp(a.ID)
 		if err != nil {
```

Comment:
any reason why this wasn't moved as well?

Reply:
woops!